topics/acl.md

```diff
@@ -268,8 +268,41 @@ the case of an ACL that is just additive, that is, in the form of `+@all -...`
 You should be absolutely sure that you'll never include what you did not mean
 to.
 
-However to remember that categories are defined, and what commands each
-category exactly includes, is impossible and would be super boring, so the
+The following is a list of command categories and their meanings:
+* keyspace - Writing or reading from keys, databases, or their metadata 
+  in a type agnostic way. Includes `DEL`, `RESTORE`, `DUMP`, `RENAME`, `EXISTS`, `DBSIZE`,
+  `KEYS`, `EXPIRE`, `TTL`, `FLUSHALL`, etc. Commands that may modify the keyspace,
+  key or metadata will also have `write` category. Commands that only read
+  the keyspace, key or metadata will have the `read` category.
+* read - Reading from keys (values or metadata). Note that commands that don't
+  interact with keys, will not have either `read` or `write`.
+* write - Writing to keys (values or metadata).
+* admin - Administrative commands. Normal applications will never need to use
+  these. Includes `REPLICAOF`, `CONFIG`, `DEBUG`, `SAVE`, `MONITOR`, `ACL`, `SHUTDOWN`, etc.
+* dangerous - Potentially dangerous commands (each should be considered with care for
+  various reasons). This includes `FLUSHALL`, `MIGRATE`, `RESTORE`, `SORT`, `KEYS`,
+  `CLIENT`, `DEBUG`, `INFO`, `CONFIG`, `SAVE`, `REPLICAOF`, etc.
+* connection - Commands affecting the connection or other connections.
+  This includes `AUTH`, `SELECT`, `COMMAND`, `CLIENT`, `ECHO`, `PING`, etc.
```

Comment:
Why is `COMMAND` here?

Reply:
this is not a comprehensive list of all the commands of each category. just a bunch of examples to demonstrate the different types of concepts that fit in each category, so in theory we can remove COMMAND from that list in the docs if we want, but it is in the category in redis.

i think the reason COMMAND is in that category is that normally client libraries call COMMAND on startup in order to get the info about `firstkey`, `lastkey`, etc, so that they can handle clusters.